argo_prod/schema.xml

```diff
@@ -8,37 +8,61 @@
 
   <fields>
     <field name="id" type="string" stored="true" indexed="true" multiValued="false" required="true"/>
+    <!-- DEPRECATED: "long" is a Trie based field type; use solr.LongPointField class instead ... -->
     <field name="_version_" type="long" indexed="true" stored="true" multiValued="false"/>
+    <!-- DEPRECATED: "date" is a Trie based field type; use solr.DatePointField class instead ... -->
     <field name="timestamp" type="date" indexed="true" stored="true" default="NOW" multiValued="false"/>
 
     <!-- NOTE:  not all possible Solr field types are represented in the dynamic fields -->
 
     <!-- string (_s...) -->
-    <dynamicField name="*_si"    type="string"    stored="false" indexed="true"  multiValued="false"/>
-    <dynamicField name="*_sim"   type="string"    stored="false" indexed="true"  multiValued="true"/>
-    <dynamicField name="*_ss"    type="string"    stored="true" indexed="false" multiValued="false"/>
-    <dynamicField name="*_ssm"   type="string"    stored="true" indexed="false" multiValued="true"/>
-    <dynamicField name="*_ssi"   type="string"    stored="true" indexed="true"  multiValued="false"/>
-    <dynamicField name="*_ssim"  type="string"    stored="true" indexed="true"  multiValued="true"/>
-    <dynamicField name="*_ssort" type="alphaSort" stored="false" indexed="true"  multiValued="false"/>
-
-    <!-- integer (_i...) -->
+    <!-- docValues true makes for faster facets, but more storage -->
+    <dynamicField name="*_si"     type="string"    stored="false" indexed="true"  multiValued="false"/>
```

Comment:
some of this is whitespace -- use the github option to ignore it:  
<img width="342" alt="image" src="https://github.com/sul-dlss/sul-solr-configs/assets/96775/e1a9a2c9-0175-4a55-b417-109c2526e8c1">